sdcm/cluster.py

```diff
@@ -1961,7 +1972,8 @@ def install_scylla(self, scylla_repo):
             self.remoter.sudo('SUSEConnect --product sle-module-python2/15.3/x86_64')
             self.remoter.sudo('zypper install -y python2-PyYAML', ignore_status=True)
             self.remoter.sudo('zypper install -y python3-PyYAML', ignore_status=True)
-            self.remoter.sudo('zypper install -y {}'.format(self.scylla_pkg()))
+            version = f"-{scylla_version}" if scylla_version else ""
```

Comment:
I wonder if we want to change this without testing ? I don't think we tes suse anymore...
(or you are trying this locally?)